src/components/annual-report/utils/utils.js

```diff
@@ -38,8 +47,20 @@ export const convertAnalyticsResponseToObject = ({ headers, rows }) => {
             restructuredData[dx][ou] = {}
         }
 
-        if (!restructuredData[dx][ou][pe] && !isNaN(Number(rowData.value))) {
-            restructuredData[dx][ou][pe] = Number(rowData.value)
+        if (!restructuredData[dx][ou][pe]) {
+            if (co) {
+                restructuredData[dx][ou][pe] = {}
```

Comment:
this has been modified to allow for an additional level for coc in the data structure (but will behave as before if there is no `co` dimension in response)